cli/deployments/eirini.go

```diff
@@ -60,7 +61,7 @@ func (k Eirini) Delete(c kubernetes.Cluster) error {
 	return c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), eiriniDeploymentID, metav1.DeleteOptions{})
 }
 
-func (k Eirini) apply(c kubernetes.Cluster, options kubernetes.InstallationOptions) error {
+func (k Eirini) apply(c kubernetes.Cluster, ui *ui.UI, options kubernetes.InstallationOptions) error {
```

Comment:
Isn't it better if we just attach the UI to the Deployment object? This way it's easier to have a global UI without the need to pass it around in all methods that need it.

Reply:
I tried to make it a field in the `Installer` structure IIRC.
That resulted in a circular import.
Not sure if that will happen for `Deployments` as well.

Reply:
Hm. `Deployment` is an interface. I guess I can add `ui` fields to the various implementations, and add a `UI` interface method with which to set that information.

Reply:
Probably a `SetUI` method

Reply:
Yes, that. To come soon, just finished the `Installer` change ...

Reply:
Done with commit 5135955e4a0db387de755ea1bf73a4f8c85658b2